buildSrc/src/main/resources/templates/pojo.mustache

```diff
@@ -46,53 +48,54 @@ import com.fasterxml.jackson.annotation.JsonValue;
   }{{/generateBuilders}}
 
   {{#vendorExtensions.x-extra-annotation}}{{{vendorExtensions.x-extra-annotation}}}{{/vendorExtensions.x-extra-annotation}}{{#useSwaggerAnnotations}}
-  @ApiModelProperty({{#example}}example = "{{{example}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}value = "{{{description}}}"){{/useSwaggerAnnotations}}
+  @ApiModelProperty({{#example}}example = "{{{.}}}", {{/example}}{{#required}}required = {{required}}, {{/required}}value = "{{{description}}}"){{/useSwaggerAnnotations}}
   @JsonProperty("{{baseName}}")
 {{#useBeanValidation}}{{>beanValidation}}{{/useBeanValidation}}  public {{{datatypeWithEnum}}} {{getter}}() {
     return {{name}};
   }
 
-  public void {{setter}}({{{datatypeWithEnum}}} {{name}}) {
+  @JsonProperty("{{baseName}}")
+  {{#vendorExtensions.x-setter-extra-annotation}}{{{vendorExtensions.x-setter-extra-annotation}}}
+  {{/vendorExtensions.x-setter-extra-annotation}}public void {{setter}}({{{datatypeWithEnum}}} {{name}}) {
     this.{{name}} = {{name}};
   }
 
   {{#isArray}}
-  public boolean add{{nameInCamelCase}}({{{items.datatypeWithEnum}}} {{name}}) {
+  public {{classname}} add{{nameInCamelCase}}Item({{{items.datatypeWithEnum}}} {{name}}Item) {
```

Comment:
This change brings up an interesting question/debate: Generally with mutators that can do nothing (as is the case when adding elements to a set or map, or removing from a collection), we return a boolean so the caller can know if the operation had an effect on the object. 

I guess there's an argument to be made that if the operation is to be considered idempotent, then the boolean output has no real value, since the add will always ensure the element is there, or fail with an exception; with remove being the inverse -- ensuring the element doesn't exist in the set. This example kind of fails in the array/list implementation of remove, since it'd be possible for "remove" to succeed on repeated calls, without fully removing the target element. For instance:

```
list = ["a", "b", "a", "c", "a"];
list.remove("a").remove("a"); // an instance of "a" still exists in the list.
```

Whether or not this matters is one of usability and expectation, I suppose. I can assume a stance from this change, but what are your thoughts on this?

Reply:
Gotcha